packages/@o3r/configuration/schematics/use-config-signal/index.ts

```diff
@@ -89,9 +89,11 @@ function ngUseConfigSignalFn(options: NgUseConfigSignalSchematicsSchema): Rule {
                   !member.name
                   || (
                     ts.isIdentifier(member.name)
-                    && member.name.escapedText !== 'config$'
-                    && member.name.escapedText !== 'config'
-                    && member.name.escapedText !== 'dynamicConfig$'
+                    && ![
```

Comment:
I am not sure that this is more performant than the `ifs`. anyway not important here